style.css

```diff
@@ -12080,6 +12080,55 @@ button + .button {
   padding-left: 24px;
 }
 
+.gform_wrapper .gform_body ul.gform_fields .gsurvey-survey-field .gsurvey-likert .nhsuk-likert {
+  border: 1px solid #768692;
```

Comment:
Should we use NHS colour variables here?

Reply:
style.css is the output, so nhs vars have been processed already. ;)

Reply:
Rookie 🤦